lightning/src/ln/channelmanager.rs

```diff
@@ -1313,6 +1323,7 @@ impl<ChanSigner: ChannelKeys, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 								commitment_signed,
 							},
 						});
+						self.persistence_notifier.notify();
```

Comment:
This needs to happen even if the update_channel call fails or in the None block (which IIRC implies that there is now a pending outbound HTLC in the holding cell).